src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -770,6 +776,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Private ReadOnly Property SyntaxTreesSet As HashSet(Of SyntaxTree)
```

Comment:
Why "SyntaxTreesSet" and "_syntaxTreeSet"? At first blush I'd expect either both or neither to be plural.